zulipterminal/ui_tools/boxes.py

```diff
@@ -366,30 +365,19 @@ def reactions_view(self, reactions: List[Dict[str, Any]]) -> Any:
         if not reactions:
             return ''
         try:
-            std_reaction_stats = Counter()  # type: typing.Counter[str]
-            custom_reaction_stats = Counter()  # type: typing.Counter[str]
+            emoji_stats = Counter()  # type: typing.Counter[str]
             for reaction in reactions:
-                if reaction['reaction_type'] == 'unicode_emoji':
-                    std_reaction_stats[reaction['emoji_code']] += 1
-                else:
-                    # Includes realm_emoji and zulip_extra_emoji
-                    custom_reaction_stats[reaction['emoji_name']] += 1
-
-            std_reactions = [
-                '{} {}'.format(
-                    '\\U{:0>8}'.format(reaction)
-                    .encode().decode('unicode-escape'),
-                    count)
-                for reaction, count in std_reaction_stats.items()
-            ]
-            custom_reactions = [
+                emoji_stats[reaction['emoji_name']] += 1
+
+            spaced_emoji_stats = [
                 ':{}: {}'.format(reaction, count)
-                for reaction, count in custom_reaction_stats.items()
+                for reaction, count in emoji_stats.items()
             ]
-            all_emoji = std_reactions + custom_reactions
+
             spaced_emoji = [
```

Comment:
I know you've not adjusted the name of this variable, but I think it'd be useful to have a consistent name here - `spaced_reaction_texts`?